ntoskrnl/mm/ARM3/expool.c

```diff
@@ -2411,6 +2420,32 @@ ExAllocatePoolWithTag(IN POOL_TYPE PoolType,
     return POOL_FREE_BLOCK(Entry);
 }
 
+#ifdef DBG_NUMBER_OF_POOL_CALLERS
+PVOID
+NTAPI
+ExAllocatePoolWithTag(IN POOL_TYPE PoolType,
```

Comment:
Since you separated this function you can just as well use the new SAL annotation (and moving the parameters to new lines).

Reply:
Fixed